irods/test/meta_test.py

```diff
@@ -338,6 +338,35 @@ def test_metadata_manipulations_with_admin_kw__364__365(self):
             if d: d.unlink(force=True)
             if user: user.remove()
 
+    def test_atomic_metadata_operations_with_admin_kw__issue_576(self):
+        ses = data = user = None
+        adm = self.sess
+
+        if adm.server_version <= (4,2,11):
+            self.skipTest('ADMIN_KW not valid for Metadata API in iRODS 4.2.11 and previous')
```

Comment:
This is true even for the atomic metadata API? I see this skip elsewhere in these tests but that's for the legacy metadata API. Did you discover this through testing?

Reply:
Ooh, found it: https://github.com/irods/irods/issues/6198 Resolving.